pmm-app/src/shared/components/Form/NumericInput/NumericInput.style.ts

```diff
@@ -0,0 +1,100 @@
+import { css } from 'emotion';
+import { GrafanaTheme } from '@grafana/data';
+import { stylesFactory } from '@grafana/ui';
+
+export const getStyles = stylesFactory((theme: GrafanaTheme) => {
+  const arrow = css`
+    position: absolute;
+    right: 0.3em;
+    width: 0;
+    height: 0;
+    border-style: solid;
+  `;
+
+  const button = css`
+    position: absolute;
+    border: none;
+    right: 2px;
+    width: 1.2em;
+    text-align: center;
+    cursor: default;
+    background: transparent;
+    z-index: 1;
+    color: ${theme.colors.formInputText};
+
+    &:disabled {
+      color: ${theme.colors.formInputDisabledText};
+      cursor: not-allowed;
+    }
+
+    &:focus {
+      outline: none;
+    }
+  `;
+
+  const baseInput = css`
+    height: 100%;
+
+    input {
+      box-sizing: border-box;
+      border-radius: 3px;
+      padding-right: 1.4em;
+    }
+
+    &,
+    &:hover {
+      input[type='number']::-webkit-outer-spin-button,
```

Comment:
I wonder what will it be like in non-webkit browsers?

Reply:
AFAIK other browsers always show spin buttons (BTW, this is the approach I found the the Input component by Grafana)